querybook/server/models/user.py

```diff
@@ -66,8 +74,12 @@ def to_dict(self, with_roles=False):
             "profile_img": self.profile_img,
             "email": self.email,
             "deleted": self.deleted,
+            "is_group": self.is_group,
         }
 
+        if self.is_group:
+            user_dict["properties"] = self.properties
```

Comment:
properties are private for users, can you make it like
user_dict['group_info'] = self.properties.get('group_info', {})

Reply:
that's why I only return properties for groups with the check of `if self.is_group:`. will there be any private property for groups as well? otherwise, I feel like it's not necessary to add additional nested layer.

Reply:
i felt it is still necessary to put it into a subfield which allows groups to have public/private info for future. Not only it makes this more flexible, but also we no longer require this if check to determine the visibility of properties field. The data will be self explanatory